docs/miners/mainnet.md

```diff
@@ -0,0 +1,120 @@
+# Mainnet Miners
+
+# **DO NOT RUN THE BASE MINER ON MAINNET!**
+
+> **The base miner provided in this repo is _not intended_ to be run on mainnet!**
+>
+> **If you run the base miner on mainnet, you are not guaranteed to earn anything!**
+> It is provided as an example to help you build your own custom models!
+>
+
+## Compute Requirements
+
+|   Miner   |
+|-----------|
+|  8gb RAM  |
+|  2 vCPUs  |
+
+## Installation
+
+First, install PM2:
+```
+sudo apt update
+sudo apt install nodejs npm
+sudo npm install pm2@latest -g
+```
+
+Verify installation:
+```
+pm2 --version
+```
+
+Clone the repository:
+```
+git clone https://github.com/foundryservices/snpOracle.git
+cd snpOracle
+```
+
+Create and source a python virtual environment:
+```
+python3 -m venv
+source .venv/bin/activate
+```
+
+Install the requirements with poetry:
+```
+pip install poetry
+poetry install
+```
+
+## Configuration
+
+#### Environment Variables
+First copy the `.env.template` file to `.env`
+
+```shell
+cp .env.template .env
+```
+
+Update the `.env` file with your miner's values.
+
+```text
+WANDB_API_KEY='REPLACE_WITH_WANDB_API_KEY'
+MINER_HF_ACCESS_TOKEN='REPLACE_WITH_HUGGINGFACE_ACCESS_KEY'
+HF_ACCESS_TOKEN='REPLACE_WITH_HUGGINGFACE_ACCESS_KEY'
```

Comment:
HF_ACCESS_TOKEN not needed for miners. They also do not need any of the git related env variables.